src/Compilers/CSharp/Test/Emit2/Semantics/PatternSwitchTests.cs

```diff
@@ -1214,7 +1215,7 @@ public static void M(object o)
 }";
             var compilation = CreateCompilation(source, options: TestOptions.DebugExe);
             compilation.VerifyDiagnostics();
-            var expectedOutput =
+            var expectedOutput = FormattableString.Invariant(
```

Comment:
Yes. The actual code in our Unit Tests still runs under the system locale. That means when running on a Spanish machine this code runs with `CultureInfo.CurrentCulture` as `es-ES` and this baseline. To ensure the baselines are generated with a known format (since we're comparing with a static baseline) this needs to be formatted with invariant culture.